tuple/include/tuple_sketch_impl.hpp

```diff
@@ -315,7 +315,7 @@ uint64_t compact_tuple_sketch<S, A>::get_theta64() const {
 
 template<typename S, typename A>
 uint32_t compact_tuple_sketch<S, A>::get_num_retained() const {
-  return entries_.size();
+  return static_cast<uint32_t>(entries_.size());
```

Comment:
We need to do this cast enough that i wonder if we should define a `getNumEntries()` method to handle it?

Reply:
I had the same thought, but did not act on it yet